.storybook/config.js

```diff
@@ -86,7 +86,7 @@ global._googlesitekitBaseData = {
 	blogPrefix: 'wp_',
 	ampMode: false,
 	isNetworkMode: false,
-	isFirstAdmin: true,
```

Comment:
This should probably be kept since it's currently included in `_googlesitekitBaseData`.